CHANGELOG.md

```diff
@@ -36,6 +36,7 @@
 ### Changed
 
 - [#5467](https://github.com/ChainSafe/forest/pull/5467) Improve error message for `Filecoin.EthEstimateGas` and `Filecoin.EthCall`.
+- [#5587](https://github.com/ChainSafe/forest/pull/5587) Retrieve state root from child tipset rather than recomputing when possible
```

Comment:
```suggestion
```

This is not really a user-facing change, so let's omit it from the changelog.

Reply:
Done.